extensions/Vendor/FRAUNHOFER_materials_pbr/example/untextured-pbr/index.html

```diff
@@ -0,0 +1,462 @@
+                vec3  F = F_Schlick( specularColor, dotLH );
+                float G = G_GGX_SmithCorrelated( alpha, dotNL, dotNV );
+                float D = D_GGX( alpha, dotNH );
+                
+                return F * (G * D);
+            }
+
+
+            /////////////////////////////////////////////////////////
+            //          MAIN LIGHTING COMPUTATION FUNCTION         //
+            /////////////////////////////////////////////////////////
+
+            void computeLighting(const in IncidentLight directLight,
+                                 const in GeometricContext geometry,
+                                 const in PhysicalMaterial material,
+                                 inout ReflectedLight reflectedLight)
+            {
+                float dotNL     = saturate(dot(geometry.normal, directLight.direction));
+                vec3 irradiance = dotNL * directLight.color;
+
+                irradiance *= PI;
+                
+                reflectedLight.specular += irradiance * BRDF_Specular_GGX(directLight,
+                                                                          geometry,
+                                                                          material.specularColor,
+                                                                          material.specularRoughness);
+
+                reflectedLight.diffuse  += irradiance * BRDF_Diffuse_Lambert(material.diffuseColor);
+            }
+
+
+            /////////////////////////////////////////////////////////
+            //                  MAIN PROGRAM                       //
+            /////////////////////////////////////////////////////////
+
+            void main()
+            {
+                // light definitions (simple directional lights)
+
+                IncidentLight directLight0;
+                directLight0.direction = vec3(0.0, 0.0, 1.0);
+                //directLight0.color     = vec3(0.7, 0.7, 0.7);
+                directLight0.color     = 0.7 * vec3(1.0, 1.0, 1.0);
+
+                IncidentLight directLight1;
+                directLight1.direction = vec3(-0.71, 0.71, 0.0);
+                //directLight1.color     = vec3(0.5, 0.5, 0.5);
+                directLight1.color     = 0.7 * vec3(0.8, 0.6, 1.0);
+
+                IncidentLight directLight2;
+                directLight2.direction = vec3(0.71, 0.71, 0.0);
+                //directLight2.color     = vec3(0.5, 0.5, 0.5);
+                directLight2.color     = 0.7 * vec3(1.0, 0.8, 0.6);
+                
+
+                // material definition
+
+                vec4 diffuseColor = vec4(diffuse, 1.0);
+
+                float roughnessFactor = roughness;
+                float metalnessFactor = metalness;
+
+                PhysicalMaterial material;                
+                material.diffuseColor      = diffuseColor.rgb * ( 1.0 - metalnessFactor );
+                material.specularRoughness = clamp( roughnessFactor, 0.04, 1.0 );
```

Comment:
Was able to track this down in three.js. Looks like it used to be some different values -https://github.com/mrdoob/three.js/blob/b1df0234204f9a43eaa3c80da91803e87ded52d8/src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl#L3 
Based on disney's remapping - http://blog.selfshadow.com/publications/s2012-shading-course/burley/s2012_pbs_disney_brdf_notes_v3.pdf 
Page 16, Specular G details: "This remapping was based on comparisons with measured data as well as artist feedback that the specular was just “too hot” for small roughness values." 

Three.js was then updated with the new mapping here - https://github.com/mrdoob/three.js/blob/5a6df635c32071bec92df393f85c0d72ba34b560/src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl#L3

I am assuming the update was for similar reasons based on artist feedback. The 0.04 value confused me too since it happens to be the same as Specular Coefficient but they seem to exist for very different reasons?